scilpy/io/utils.py

```diff
@@ -265,3 +266,25 @@ def read_info_from_mb_bdo(filename):
     radius = np.asarray(radius, dtype=np.float32)
     center = np.asarray(center, dtype=np.float32)
     return geometry, radius, center
+
+
+def load_matrix_in_any_format(filepath):
+    _, ext = os.path.splitext(filepath)
+    if ext == '.txt':
+        data = np.loadtxt(filepath)
+    elif ext == '.npy':
+        data = np.load(filepath)
+    else:
+        raise ValueError('Extension {} is not supported'.format(ext))
+
+    return data
+
+
+def save_matrix_in_any_format(filepath, output_data):
```

Comment:
You can save without extension but then you can't read it ?

Reply:
Well for saving I put a default type, but for loading I didn't want to do try/catch and guessing the kind of file format

Reply:
default type but without extension so it won't be read with load_matrix_in_any_format.

Reply:
Ahhh, 1000 times sorry. I though I was adding the extension in the else...